implementations/micrometer-registry-new-relic/build.gradle

```diff
@@ -2,6 +2,11 @@ dependencies {
     api project(':micrometer-core')
 
     implementation 'org.slf4j:slf4j-api'
+    implementation 'com.newrelic.agent.java:newrelic-api'
```

Comment:
Adding APM support requires the addition of a compile time dependency on the newrelic-api.
The API is fairly light, containing only annotations, interfaces, POJOs, and enums, so I don't expect this to cause any issues for consumers not using the APM integration